kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -247,6 +360,18 @@
       copySettings() {
         this.settingsCopy = Object.assign({}, this.settings);
       },
+      handleCreatePin() {
```

Comment:
I believe the issue @pcenov noted about not being able to save the facility name is due to the call to `this.saveFacilityName` in `FacilityConfigPage.methods.sendFacilityName` where it calls that method that is not in scope. As that `saveFacilityName`  is set as an action in the `facilityConfig` Vuex module, perhaps we just need to `...mapActions({ saveFacilityName: 'facilityConfig/saveFacilityName'})` in the methods there to provide `this.saveFacilityName` as a function.

Reply:
@nucleogenesis the failure to save has been fixed with https://github.com/learningequality/kolibri/pull/10144. Once review is complete and the pr merged, the issue will be fixed on feature branch too! cc @pcenov

Reply:
@akolson @pcenov - https://github.com/learningequality/kolibri/pull/10144 reviewed and merged ✅